index.html

```diff
@@ -0,0 +1,269 @@
+      </div>
+    </section>
+
+    <section class='project project2'>
+      <div class="img-container">
+        <img src="#" alt="" />
+      </div>
+      <div class="img-footer-container">
+        <h2>Multi-post Stories Gain+Glory</h2>
+        <ul>
+          <li>Python</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class='project project3'>
+      <div class="img-container">
+        <img src="#" alt="" />
+      </div>
+      <div class="img-footer-container">
+        <h2>Multi-post Stories Gain+Glory</h2>
+        <ul>
+          <li>Python</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class='project project4'>
+      <div class="img-container">
+        <img src="#" alt="" />
+      </div>
+      <div class="img-footer-container">
+        <h2>Multi-post Stories Gain+Glory</h2>
+        <ul>
+          <li>Python</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class='project project5'>
+      <div class="img-container">
+        <img src="#" alt="" />
+      </div>
+      <div class="img-footer-container">
+        <h2>Multi-post Stories Gain+Glory</h2>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class='project project6'>
+      <div class="img-container">
+        <img src="#" alt="" />
+      </div>
+      <div class="img-footer-container">
+        <h2>Multi-post Stories Gain+Glory</h2>
+        <ul>
+          <li>Python</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+  </section>
+
+  <section class="about-me-container">
+    <div class="background1-aboutme">
+
+    </div>
+    <div class="background2-aboutme">
+
+    </div>
+    <section class="about-me">
+      <h1>About me</h1>
+      <p class='about-paragraph'>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what
+        you see and have a project you need coded,
+        don’t hestiate to contact me.
+      </p>
+      <button>Get My Resume</button>
+    </section>
+
+    <section class="habilities">
+      <div class="languages">
+        <img src="assets/icon-languages.png" alt="">
+        <h2>Languages</h2>
+        <ul>
+          <li>JavaScript</li>
+          <li>Python</li>
+          <li>Html</li>
+          <li>CSS</li>
+        </ul>
+
+      </div>
+      <div class="frameworks">
```

Comment:
-  [ ] As suggested early, Kindly make sure all the about part is grabbed in one `section` tag